test/integration/testcases/alternate-hostname.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html><head>
+    <title>Alternate hostname testing</title>
+</head>
+<body>
+  <div>
+    <p>
+      If alternate hostname support is working, there should be an svg image of the vue logo below.
+    </p>
+    <p>
+      It works because localhost.me resolves to 127.0.0.1, allowing us to serve the vue logo over
+      it, and we start the integration tests by specifying localhost.me as an allowed-hostname
+      with the -h flag.
+    </p>
+    <img src="http://localtest.me:8000/vue.svg">
```

Comment:
Curious how this works in CI? Are we editing a host file to make this work?

Reply:
@Robdel12 no, `localtest.me`'s dns entry points to 127.0.0.1.  So in CI the node server we spin up that listens to 127.0.0.01:8000 works just the same way it does locally, with no configuration needed.